src/mxarray.jl

```diff
@@ -655,10 +655,10 @@ function jstring(mx::MxArray)
         throw(ArgumentError("jstring only applies to strings (i.e. char vectors)."))
     end
     len = ncols(mx) + 2
-    tmp = Array(UInt8, len)
+    tmp = Array(UInt8, len-2)
     ccall(_mx_get_string, Cint, (Ptr{Void}, Ptr{UInt8}, mwSize), 
         mx.ptr, tmp, len)
-    bytestring(pointer(tmp))
+    String(tmp)
```

Comment:
Thanks fixed this.  For the life of me I can't get the tests to pass due to a random error unrelated to the tests 

```julia
 A MATLAB session is open successfully
Internal Error: A primary message table for module 77 was already registered with a differing collection of error messages.Severe:
The program '[17396] C:\Julia\Julia-0.4.6\bin\julia.exe: Native' has exited with code 1 (0x1).
======================================================================[ ERROR: MATLAB ]======================================================================


failed process: Process(`'C:\Julia\Julia-0.4.6\bin\julia' --check-bounds=yes --code-coverage=none --color=yes 'C:\Users\Mustafa\.julia\v0.4\MATLAB\test\runtests.jl'`, ProcessExited(1)) [1]

=============================================================================================================================================================

ERROR: MATLAB had test errors
 in test at pkg/entry.jl:803
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:32
 in cd at pkg/dir.jl:31
 in test at pkg.jl:71

```

Anyone care to test before we merge this?